### PR TITLE
buildplatform

### PR DESCRIPTION
**Pull request by kaovilai**
- **BUILDPLATFORM**

## Why the changes were made
Speed when building for different arch

## How to test the changes made

`docker build --platform=linux/<not-local-arch> .`

### CODE REVIEW

File: must-gather/Dockerfile
Comment (kaovilai) [inline must-gather/Dockerfile:1]: need to add goarch to build commands..

**Review by sseago — DISMISSED** [review]

**Review by mateusoliveira43 — APPROVED** [review]
images job passed

**Review by weshayutin — APPROVED** [review]
/LGTM

**Pull request merged**